sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/commands.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
So are we extracting some of DDL relative plans in this `commands.scala`? But there're still other commands in `basicLogicalOperators.scala`. What's the relation of the two?

Reply:
I'd like to put all the DDL/DML commands in this file, starting with DELETE/UPDATE

Reply:
Actually, let me do it together in an individual PR.